server/features/users/Mutation.ts

```diff
@@ -106,11 +109,62 @@ export const UserMutation = extendType({
       }
     });
 
+    t.field('updateUser', {
+      type: nullable('User'),
+      args: {
+        id: 'ID',
+        email: 'String',
+        role: 'UserRole'
+      },
+      authorize: guard(
+        or(current(), authorized(UserRole.ADMIN)),
+        validated(UpdateUser)
+      ),
+      resolve: (parent, { id, email, role }, { db }) => db.user.update({
+        where: { id },
+        data: {
+          email,
+          role
```

Comment:
Allows normal users to make themselves admin

Reply:
ok